Doc/library/enum.rst

```diff
@@ -730,8 +730,7 @@ Some rules:
 2. While :class:`Enum` can have members of any type, once you mix in an
    additional type, all the members must have values of that type, e.g.
    :class:`int` above.  This restriction does not apply to mix-ins which only
-   add methods and don't specify another data type such as :class:`int` or
-   :class:`str`.
+   add methods and don't specify another data type.
```

Comment:
Instead of removing the `int` and `str` clause, make it a "for example" clause; i.e.:
```
... another data type; for example :class:`int` or :class:`str`.
```

Reply:
Actually I believe there is a mistake here.

If I understand correctly, once you mix a type that specifies a data type (_int_, _float_ or whatever), all members must be of that type. There is no such restriction for mixins without constructor that only add "pure methods", which is NOT the case for _int_ and _str_. See https://pastebin.com/qeb4npZw for example.

And the doc seems to say that _int_ and _str_ only add methods without defining types, which, I believe, is incorrect ; hence I've removed the example.

Reply:
Ah, I see it now.  The extra (supposed to be) clarifying example could actually be confusing instead.

Okay, keep this change as-is.